### PR TITLE
Adds Tests for SubText

### PR DESCRIPTION
**Pull request by LeonardKrml**
### Please check if the PR fulfills these requirements
- [x] I made sure that my code builds
- [x] I merged the master into this branch before pushing
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?** (check one with "x")
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

### What is the current behavior? (You can also link to an open issue here)
#465 

### What is the new behavior?
renamed parameter ent to len, added test for SubText

### Does this PR introduce a breaking change? (check one with "x")
- [ ] Yes
- [x] No

###  Other information

### CODE REVIEW

File: src/Yaapii.Atoms/Text/SubText.cs [hunk 1]
```diff
@@ -43,11 +43,11 @@ public SubText(String text, int strt) : this(new LiveText(text), strt)
         /// </summary>
         /// <param name="text">text to extract from</param>
         /// <param name="strt">where to start</param>
-        /// <param name="end">where to end</param>
-        public SubText(String text, int strt, int end) : this(
+        /// <param name="len">length of substring</param>
+        public SubText(String text, int strt, int len) : this(
```
Comment (icaOBU) [inline src/Yaapii.Atoms/Text/SubText.cs:47]: Please adjust the parameter names so we can see for what the parameter is used. So you can remove the parameter description from header. Most programmers forget to adjust these comments and the parametrer name should say what the param stands for. 

Also please don't use abbreviations. I think this is old code and we should avoid something like that

**Review by icaOBU — CHANGES_REQUESTED** [review]

**Review by icaOBU — APPROVED** [review]

File: tests/Yaapii.Atoms.Tests/Text/SubTextTest.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+
+        [Fact]
+        public void CutStringwithLength()
+        {
+            Assert.Equal(
+                "the",
+                new SubText(
+                    "this_is:the_end",
+                    8,3
```
Comment (aveXcaesar) [inline tests/Yaapii.Atoms.Tests/Text/SubTextTest.cs:51]:
```suggestion
                    8,
                    3
```

File: tests/Yaapii.Atoms.Tests/Text/SubTextTest.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        }
+
+        [Fact]
+        public void CutITextwithLength()
+        {
+            Assert.Equal(
+                "the",
+                new SubText(
+                    new LiveText("this_is:the_end"),
+                    8, 3
```
Comment (aveXcaesar) [inline tests/Yaapii.Atoms.Tests/Text/SubTextTest.cs:75]:
```suggestion
                    8,
                    3
```

**Review by aveXcaesar — CHANGES_REQUESTED** [review]
Only styling ;-)

**Review by aveXcaesar — APPROVED** [review]

**Review by jaanpeeter — APPROVED** [review]

**Pull request merged**